hypertools/plot/plot.py

```diff
@@ -171,12 +192,18 @@ def plot(x, fmt=None, marker=None, markers=None, linestyle=None,
             warnings.warn('Both marker and markers defined: marker will be \
                           ignored in favor of markers.')
 
+    # handle legend
+    if legend is not None:
+        mpl_kwargs['label'] = legend
+
     # normalize
     x = normalizer(x, normalize=normalize, internal=True)
```

Comment:
I think we want the normalizer step before any dimensionality reduction and alignment.  The reason is that if your features are scaled differently, there variances will also be different, which will influence the PCA weights: https://stats.stackexchange.com/questions/69157/why-do-we-need-to-normalize-data-before-analysis